Cargo.toml

```diff
@@ -39,9 +38,12 @@ serde = { version = "1", features = ["derive"] }
 structopt = "0.3"
 structopt-derive = "0.4"
 surf = "2"
-tide = { version = "0.16.0", features = ["unstable"] }
+tide = { version = "0.16.0", default-features = false, features = ["h1-server", "sessions", "unstable"] }
 tide-websockets = "0.3.0"
 toml = "0.5"
+tracing = "0.1.25"
+tracing-futures = "0.2.5"
```

Comment:
This crate is not used anywhere.
Plus, the `tracing` crate added the `Instrument` and `Instrumented` traits directly so this that we probably don't need `tracing-futures` in the future.

https://github.com/tokio-rs/tracing/blob/master/tracing/CHANGELOG.md

Reply:
The `tracing-futures` crate is needed for instrument `async` functions. Are you saying that this has just recently changed? Without `tracing-futures` present, we will get a compilation error.

Reply:
Since version `0.1.21` of the main `tracing` crate the [`Instrument`](https://docs.rs/tracing/0.1.25/tracing/trait.Instrument.html) trait was moved into it. Therefore, we can instrument `Future`s without depending on `tracing-futures`.

I removed `tracing-futures` and could build without problems.

Reply:
Copy that. I'll drop the dependency then.